tests/testthat/test-helpers.R

```diff
@@ -65,11 +65,11 @@ test_that("use_dependency() upgrades a dependency", {
   withr::local_options(list(usethis.quiet = FALSE, crayon.enabled = FALSE))
 
   expect_message(use_dependency("usethis", "Suggests"))
-  expect_match(desc::desc_get("Suggests", proj_get()), "usethis")
+  expect_match(desc::desc_get("Suggests"), "usethis")
```

Comment:
Using desc without specific arguments works in tests because `local_package()`/`local_project()` set the working directory. I don't love this interface but converting everything to `proj_desc()$get()` etc didn't seem like a big win to me.